src/utils/daos_hdlr.c

```diff
@@ -2487,8 +2218,7 @@ fs_copy_file(struct cmd_args_s *ap,
 		ssize_t bytes_read = file_read(ap, src_file_dfs, src_path,
 					       buf, left_to_read);
 		if (bytes_read < 0) {
-			rc = EIO;
-			DH_PERROR_SYS(ap, rc, "read 'failed on '%s'", src_path);
+			rc = daos_errno2der(rc);
```

Comment:
There is no rc from this function. Maybe these file_* functions should have signatures similar to the dfs_sys versions? I.e. bytes_read is passed/returned by reference and rc is returned from the function.
Then you would have the corresponding rc